teuthology/misc.py

```diff
@@ -1345,3 +1345,18 @@ def sh(command, log_limit=1024, cwd=None, env=None):
             output=output
         )
     return output
+
+
+def add_remote_path(ctx, local_dir, remote_dir):
+    """
+    Add key/value pair (local_dir: remote_dir) to job's info.yaml.
+    These key/value pairs are read to archive them in case of job timeout.
+    """
+    with open(os.path.join(ctx.archive, 'info.yaml'), 'r+') as info_file:
+        info_yaml = yaml.safe_load(info_file)
+        info_file.seek(0)
+        if 'archive' not in info_yaml:
+            info_yaml['archive'] = {local_dir: remote_dir}
+        else:
+            info_yaml['archive'][local_dir] = remote_dir
```

Comment:
please transpose these two branches, like

```suggestion
        if 'archive' in info_yaml:
            info_yaml['archive'][local_dir] = remote_dir
        else:
            info_yaml['archive'] = {local_dir: remote_dir}
```

Reply:
I'm discovering new github features, if PR owner press "commit suggestion" it will merge or add another commit to the PR? What is the difference to "Add suggestion to batch"?
@tchaikov how do you add such kind of suggestions?

Reply:
@tchaikov done!

Reply:
<img width="215" alt="image" src="https://user-images.githubusercontent.com/59071/91376383-80d93380-e84f-11ea-96d1-3a2c8f3ca890.png">